cli/flags.rs

```diff
@@ -376,13 +383,24 @@ To change installation directory use -d/--dir flag
           // this is a fake subcommand - it's used in conjunction with
           // AppSettings:AllowExternalSubcommand to treat it as an
           // entry point script
-          SubCommand::with_name("<script>").about("Script URL"),
+          SubCommand::with_name("[SCRIPT]").about("Script URL"),
         ),
     ).subcommand(
+      SubCommand::with_name("completions")
+        .settings(&[
+          AppSettings::DisableHelpSubcommand,
```

Comment:
It would be good to have some help text

Reply:
also an entry in the manual

Reply:
There is help, it's just disabling `deno completions help` command